apps/remix-ide/src/app.js

```diff
@@ -500,4 +505,30 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   framingService.start(params)
 
   if (params.embed) framingService.embed()
+  if (!params) {
+    if (!localStorage.getItem('hadTour_initial')) {
+      introJs().setOptions({
```

Comment:
can the execution `introJs().setOptions` and the definition of this tour (and possibly others) be put in another files?
That way it will ease the process of adding other tours in the future